### DISCUSSION TITLE
v2021.08.2_23.2_x86-64_0.34

### DISCUSSION THREAD

**Discussion by PartialVolume** [PartialVolume/shredos.x86_64#136] [Announcements]
v2021.08.2_23.2_x86-64_0.34

| Buildroot Version | Kernel Version | Shredos Minor version | Architecture | Nwipe Version |
| ----------------------- | ------------------- | ------------------------------- | ---------------- | ------------------- |
| 2021.08.2             | 5.13.19           |  023.2                            | x86-64         | 0.34                |

Other software versions used within ShredOS for disc related operations
| hexedit | hdparm | Smartctl |
| ---------- | ---------- | -------------------- |
|    1.5     |   9.62    |           7.2          |

## Notable changes in this release - ShredOS
- Added X2APIC support for servers that require this feature
such as Dell PowerEdge R720, Dell R740xd2, R440, if you could
report whether these now boot without the kernel panic. Thanks.
- Added Numachip support
- Added ScaleMP vSMP
- Added Locale, locale -a now works.
- Added en_US.UTF-8 as required by tmux
- Added tmux to allow the creation of multiple terminals viewable
simultaneously.
- Added the command tmux4 which creates a 4 panes (terminals) in the
virtual terminal. This allows multiple nwipes /dev/sdx to be run
simultaneously for those that like to run one wipe in each terminal
for hot swapping.
- Added DirectFB drivers for ATI128, Matrox and I830.
- Added ascii_invaders for something to do while those discs wipe.
- Increased size of FAT32 to 130MB with about 14MB free for PDF
wipe reports and logs.
- Bumped minor version 23.1 to 23.2

## Notable changes in this release - Nwipe
No changes. There will be lots of nwipe changes in the v0.35 release. HPA/DCO detection, PDF reports, ATA secure erase.

## Note
- The .img file for writing to a USB flash drive is both legacy (bios) and UEFI bootable.

As this is only a preview for testing booting on previously non bootable devices I'm not planning on creating .iso's or 32bit versions, unless somebody specifically has a system that for some reason they cannot boot using USB.

## Download Counters
| Architecture/format | Download link |
| ------------------------- | --------------------|
|  x86-64 .img for USB | [![GitHub release (by file)](https://img.shields.io/github/downloads/PartialVolume/shredos.x86_64/v2021.08.2_23.2_x86-64_0.34/shredos-2021.08.2_23.2_x86-64_0.34_20230320.img)](https://github.com/PartialVolume/shredos.x86_64/releases/download/v2021.08.2_23.2_x86-64_0.34/shredos-2021.08.2_23.2_x86-64_0.34_20230320.img) |

### 64bit .img for modern Intel & compatible processors post Pentium 4.
```
sha1  157f5a8a18d7eaff715e71fa7daba1d81cc3d775  shredos-2021.08.2_23.2_x86-64_0.34_20230323.img
```

![image](https://user-images.githubusercontent.com/22084881/227371811-5fa3f459-452f-449e-b29a-7ccb532c7338.png)

<hr /><em>This discussion was created from the release <a href='https://github.com/PartialVolume/shredos.x86_64/releases/tag/v2021.08.2_23.2_x86-64_0.34'>v2021.08.2_23.2_x86-64_0.34</a>.</em>

**Comment by CoodyCZ** [comment]
@PartialVolume Will the release 2021.08.2_23.2_x86-64_0.34 be available in ISO format with UEFI support?

**Reply by PartialVolume** [reply]
@CoodyCZ I'll see what I can do, no promises as I'm still working on the nwipe code for HPA/DCO and secure erase.
